library/core/tests/iter/sources.rs

```diff
@@ -1,5 +1,4 @@
 use super::*;
-use core::iter::*;
```

Comment:
Could you move all the import removals to a separate commit with a description saying that the removed imports are redundant?